lstchain/scripts/lstchain_r0_to_r0g.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+"""
+Script to apply gain selection to the raw fits.fz produced by EvB6 in CTAR1
+format (aka R1v1).
+
+Gain selection is only applied to shower events (not to interleaved pedestal
+and flatfield)
+
+It uses heuristic identification of the interleaved flatfield (FF) events (
+given the occasional problems we have with the FF event tagging)
```

Comment:
Should this be an option? Do we really want to force the detection heuristic, that e.g. doesn't work I think in moonlight conditions, given that it is supposed to be working now?

Reply:
I think it works also for moon conditions (the calibration charge is pretty high). But indeed, we should optionally trust the event_type, or perhaps even do a check of how many of the tagged FF fulfill the heuristics or vice versa. Will look into it.

Reply:
Now the heuristic identification can be disabled.